fdbcli/MetaclusterCommands.actor.cpp

```diff
@@ -135,7 +155,8 @@ ACTOR Future<bool> metaclusterDecommissionCommand(Reference<IDatabase> db, std::
 ACTOR Future<bool> metaclusterRegisterCommand(Reference<IDatabase> db, std::vector<StringRef> tokens) {
 	if (tokens.size() < 4) {
 		fmt::print("Usage: metacluster register <NAME> connection_string=<CONNECTION_STRING>\n"
-		           "[max_tenant_groups=<NUM_GROUPS>]\n\n");
+		           "[max_tenant_groups=<NUM_GROUPS>]\n"
+		           "[auto_tenant_assignment=<enabled|disabled>]\n\n");
```

Comment:
I think the usual syntax we use for this would look something like:

```suggestion
		           "[max_tenant_groups=<NUM_GROUPS> | auto_tenant_assignment=<enabled|disabled>] ...\n\n");
```